### PR TITLE
Change Time Extention or Other to Multi-select

### PR DESCRIPTION
**Pull request by dhaselhan**
* User can now select either or both, one must be selected
* Update form control to use multi select instead of toggle group
* Add new column is_other to track new boolean
* Add more tests to application controller to improve coverage

### CODE REVIEW

File: alcs-api/src/common/automapper/application-decision.automapper.profile.ts [hunk 1]
```diff
@@ -96,6 +96,44 @@ export class ApplicationDecisionProfile extends AutomapperProfile {
...
+          mapFrom((a) => a.isTimeExtension),
+        ),
+        forMember(
+          (ad) => ad.auditDate,
+          mapFrom((a) => (a.auditDate ? a.auditDate.getTime() : null)),
+        ),
+        forMember(
+          (ad) => ad.chairReviewDate,
+          mapFrom((a) =>
+            a.chairReviewDate ? a.chairReviewDate.getTime() : null,
```
Comment (mhuseinov) [inline alcs-api/src/common/automapper/application-decision.automapper.profile.ts:134]: I thought we had a helper functions for this

File: alcs-frontend/src/app/features/application/decision/decision-dialog/decision-dialog.component.ts [hunk 2]
```diff
@@ -241,12 +182,25 @@ export class DecisionDialogComponent implements OnInit {
...
         ceoCriterion: null,
+        criterionModification: [],
       });
       this.form.controls['ceoCriterion'].clearValidators();
     }
     this.form.controls['ceoCriterion'].updateValueAndValidity();
   }
 
+  onSelectCeoCriterion(ceoCriterion: CeoCriterionDto) {
+    if (ceoCriterion.code === 'MODI') {
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/application/decision/decision-dialog/decision-dialog.component.ts:193]: enum?

File: alcs-api/src/common/automapper/application-decision.automapper.profile.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-api/src/common/automapper/application-decision.automapper.profile.ts:134]: Is there? What is it called? I know of formatIncomingDate but thats for incoming not outgoing.

File: alcs-frontend/src/app/features/application/decision/decision-dialog/decision-dialog.component.ts (hunk 2 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/application/decision/decision-dialog/decision-dialog.component.ts]: Done!

**Review by mhuseinov — APPROVED** [review]

**Pull request merged**